src/http.c

```diff
@@ -1488,6 +1597,32 @@ http_response_normal(struct http_request *req, struct connection *c,
 	}
 
 	if (req != NULL) {
+		ckbuf = kore_buf_alloc(HTTP_HEADER_BUFSIZE);
+		TAILQ_FOREACH(ck, &(req->resp_cookies), list) {
+			kore_buf_reset(ckbuf);
+			kore_buf_appendf(ckbuf, "%s=%s", ck->name, ck->value);
+			if (ck->path != NULL)
+				kore_buf_appendf(ckbuf, "; Path=%s", ck->path);
+			if (ck->domain != NULL)
+				kore_buf_appendf(ckbuf, "; Domain=%s", ck->domain);
+			if (ck->expires > 0) {
+				strftime(expires, HTTP_DATE_MAXSIZE, "%a, %d %b %y %H:%M:%S GMT",
```

Comment:
Additionally I'd like to add that using HTTP_DATE_MAXSIZE here is not obvious it relates to the size of expires.

Since it is in scope sizeof(expires) as the length is much more readable and removes any doubt that it is the length of the expires array.